src/internal/model.go

```
@@ -22,20 +22,22 @@ import (
-
-var channel = make(chan channelMessage, 1000)
-var progressBarLastRenderTime time.Time = time.Now()
+// These represent model's state information, its not a global preperty
+var LastTimeCursorMove = [2]int{int(time.Now().UnixMicro()), 0} //nolint: gochecknoglobals // Todo : Move to model struct
+var ListeningMessage = true                                     //nolint: gochecknoglobals // Todo : Move to model struct
+var firstUse = false                                            //nolint: gochecknoglobals // Todo : Move to model struct
+var hasTrash = true                                             //nolint: gochecknoglobals // Todo : Move to model struct
+var batCmd = ""                                                 //nolint: gochecknoglobals // Todo : Move to model struct
+var et *exiftool.Exiftool                                       //nolint: gochecknoglobals // Todo : Move to model struct
+var channel = make(chan channelMessage, 1000)                   //nolint: gochecknoglobals // Todo : Move to model struct
+var progressBarLastRenderTime time.Time = time.Now()            //nolint: gochecknoglobals // Todo : Move to model struct
```

golangci-lint (1.64.8)

[warning] var-declaration: should omit type time.Time from declaration of var progressBarLastRenderTime; it will be inferred from the right-hand side

(revive)